ui/apps/platform/src/Containers/VulnMgmt/RiskAcceptance/ObservedCVEs/ObservedCVEsTable.tsx

```diff
@@ -279,7 +282,15 @@ function ObservedCVEsTable({
                                         <CVSSScoreLabel cvss={row.cvss} />
                                     </Td>
                                     <Td dataLabel="Affected components">
-                                        <AffectedComponentsButton components={row.components} />
+                                        <Button
+                                            variant={ButtonVariant.link}
```

Comment:
I noticed we can do 
```
variant={ButtonVariant.link}
```
or 
```
variant="link"
```
What are the pros of using the button variant instead of a string?

Reply:
While I doubt PF ever makes a breaking chain where `ButtonVariant.link` no longer === "link", I'm just trying to get in the habit of using the provided constants, because not all cases are as clear as this one.